browser/widevine/BUILD.gn

```diff
@@ -84,3 +88,66 @@ source_set("browser_tests") {
     "//url",
   ]
 }
+
+if (is_win) {
+  sign("signed_brave_exe") {
+    sources = [ "$root_out_dir/brave.exe" ]
+    outputs = [ "$root_out_dir/signed_binaries/brave.exe" ]
```

Comment:
do we sign these twice during the build? first time in `SignAndCopyPreSignedBinaries` and second time here?

Reply:
(And in short, yes, `signature_generator.py` still needs Python 2 with the `cryptography` library pre-installed. It's a constraint that comes from our devops code.)

Reply:
Thanks. got it. wrapper runs with python3.

Reply:
I forgot that I also struggled with that constraint on Windows :)

Reply:
I created a [devops issue](https://github.com/brave/devops/issues/9615) to fix this pain.